src/index.css

```diff
@@ -72,41 +72,54 @@ main {
   padding: 2vw 0;
 }
 
-.main-header {
+.main-header,
+.footer, .fine-print {
   padding: 1rem 0;
 }
 
-.main-nav {
+.main-nav,
+.footer {
   margin-top: 1em;
 }
 
-.main-nav-list {
+.main-nav-list,
+.social-media-links,
+.privacy-links {
   align-items: center;
   display: flex;
-  flex-direction: column;
-  justify-content: space-evenly;
   list-style: none;
   margin: 0;
   padding: 0;
 }
 
-.main-nav-item {
+.main-nav-list {
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+.main-nav-item,
+.privacy-link {
   font-size: 0.9em;
-  text-decoration: none;
 }
 
-.main-nav-item a {
+.main-nav-item a,
+.social-media-link a,
+.privacy-link a {
   color: inherit;
   text-decoration: none;
 }
 
-.main-nav-item a:hover {
+.main-nav-item a:hover,
+.social-media-link a:hover,
+.privacy-link a:hover {
   text-align: white;
   text-decoration: underline;
   text-decoration-skip-ink: auto;
 }
 
-.main-nav-item a:focus {
+.main-nav-item a:focus,
```

Comment:
Since these are a bunch of different classes that use the same rule, could the HTML be simplifed to use one rule instead of 3-4 duplicate ones?